stdlib/src/utils/static_tuple.mojo

```diff
@@ -404,3 +405,15 @@ struct InlineArray[ElementType: CollectionElement, size: Int](Sized):
             normalized_idx += size
 
         return self[]._get_reference_unsafe(normalized_idx)
+
+    @always_inline
+    fn unsafe_ptr(
+        self: Reference[Self, _, _]
+    ) -> UnsafePointer[Self.ElementType]:
+        """Returns a reference to the data. Use with caution.
+
+        Returns:
+            A pointer to the data.
+        """
+        # YOLO, we're crazy people. This is highly unsafe.
```

Comment:
Please remove this comment.

Reply:
Haha, sure, this is in draft, don't worry, I wouldn't let that type of comment go into the main branch